AcademyChallenge/Domain/Avatar.swift

```diff
@@ -0,0 +1,16 @@
+//
+//  Avatar.swift
+//  AcademyChallenge
+//
+//  Created by Keven Esmael on 03/10/2022.
+//
+import UIKit
+import Foundation
+
+
+struct Avatar: Codable {
+    let login: String
+	let id: Int
+	let avatar_url: URL
```

Comment:
camelCase > snake_case (see the last comment)

Reply:
solved